### PR TITLE
Get AQT configs in `aqt_compile.ipynb`

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: examples/aqt_compile.ipynb [hunk 1]
```diff
@@ -42,6 +43,17 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d5cdbf69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "directory = os.getcwd()\n",
```
Comment (richrines1) [inline examples/aqt_compile.ipynb:53]: is this necessary? i'd expect `service.aqt_get_configs("Pulses.yaml", "Variables.yaml")` to write to the current working directory already

**Comment by vtomole** [comment]
It is not. Just removed.

**Review by richrines1 — APPROVED** [review]
lgtm

**Comment by vtomole** [comment]
Tested. @richrines1 will need official approval to merge.

**Review by richrines1 — APPROVED** [review]
lgtm

**Pull request merged**